zip-0316.rst

```diff
@@ -229,22 +240,34 @@ present.
 The Unified String Encoding is resilient against typos, transcription
 errors, cut-and-paste errors, truncation, or other likely UX hazards.
 
+The Unified String Encoding is resistant, as far as reasonably possible,
+to “malleability attacks” that attempt to substitute a visually similar
+string representing a different valid Address in order to lure a user
+into sending funds to the wrong Address. In particular, as long as a user
+checks sufficiently many characters of a given Unified String Encoding
+(say, at least 20 characters) against the corresponding characters of
+a known-good reference Unified String Encoding, that check should be
+sufficient to give them confidence that a malleability attack would be
+infeasible: either they have an invalid Address Encoding, or one
+identical to the reference.
+
 There is a well-defined Unified QR Encoding of a Unified Address (or
 UFVK or UIVK) as a QR code, which produces QR codes that are reasonably
 compact and robust.
 
-There is a well-defined transformation between the Unified QR Encoding
-and Unified String Encoding of a given UA/UVK in either direction.
+There is a well-defined Unified Raw Encoding of a UA/UVK which is a
+compact byte sequence, without excessive encoding overhead. Since
+the Unified Raw Encoding is intended to only be used internally to
+Zcash-related software or transmitted over reliable channels, it need
+not have the resilience and nonmalleability properties described above.
+
+There are well-defined transformations between the Unified QR Encoding,
+Unified String Encoding, and Unified Raw Encoding of a given UA/UVK in
+any direction.
```

Comment:
Separately say that there is a UQRE <-> USE correspondence and a USE <-> URE correspondence.